### DISCUSSION TITLE
Set up a Update Library to prevent old system‘s update be removed from Windows Update

### DISCUSSION THREAD

**Discussion by MyBlueHorizon** [LegacyUpdate/LegacyUpdate#103] [Ideas]
Set up a Update Library to prevent old system‘s update be removed from Windows Update

**Comment by ghost** [comment]
Please explain! How do I do that?

**Reply by MyBlueHorizon** [reply]
Collect updates from Windows Update, build a manifest, and better to collect other release information if possible.
The Windows 9x's update has couldn't been found on Microsoft Website. Maybe Windows XP is the next one
